cachalot/tests/read.py

```diff
@@ -328,13 +338,13 @@ def test_subquery(self):
     def test_custom_subquery(self):
         tests = Test.objects.filter(permission=OuterRef('pk')).values('name')
         qs = Permission.objects.annotate(first_permission=Subquery(tests[:1]))
-        self.assert_tables(qs, Permission, Test)
+        self.assert_tables(qs, Permission, Test, ContentType)
```

Comment:
The final SQL includes an `ORDER BY "django_content_type"."app_label" ASC` 
https://github.com/django/django/blob/ca9872905559026af82000e46cde6f7dedc897b6/django/contrib/auth/models.py#L72

Thus it’s correct behavior to add `ContentType` as involved table.